### PR TITLE
[GPII-3642]: Add securitycenter.viewer role to cloud-admin account

### PR DESCRIPTION
**Pull request by natarajaya**
These two roles required to manage Security Command Center and security sources.

### CODE REVIEW

**Comment by mrtyler** [comment]
LGTM

**Comment by stepanstipl** [comment]
I have the same concern as @amatas about these roles perhaps being too wide for normal day-to-day ops? What exactly are these needed for?

**Comment by natarajaya** [comment]
@amatas @stepanstipl `Service Account Administrator` role is required to [manage security sources](https://cloud.google.com/security-command-center/docs/how-to-security-sources). 
I don't think it is one of the typical day-to-day actions – do you think it makes sense to add it to `grant_owner_role` task instead?

**Comment by stepanstipl** [comment]
> @amatas @stepanstipl `Service Account Administrator` role is required to [manage security sources](https://cloud.google.com/security-command-center/docs/how-to-security-sources).
> I don't think it is one of the typical day-to-day actions – do you think it makes sense to add it to `grant_owner_role` task instead?

That sounds good to me! 👍 

What about the `roles/iam.serviceAccountAdmin`?

**Comment by amatas** [comment]
> @amatas @stepanstipl `Service Account Administrator` role is required to [manage security sources](https://cloud.google.com/security-command-center/docs/how-to-security-sources).
> I don't think it is one of the typical day-to-day actions – do you think it makes sense to add it to `grant_owner_role` task instead?

the `grant_owner_role` sets the `owner` role for the user in the project selected, the permissions that `Service Account Administrator` owns are also included, so I guess that it's not needed to add it explicitly to the rake task.

**Comment by natarajaya** [comment]
> What about the roles/securitycenter.admin

@stepanstipl `SCC Admin` role needs to be set on organization level for anybody who needs to view  resources and change settings of SCC. As alternative we can change it to read-only role – `SCC Admin Viewer`, but in this case we would have to make a way to grant full permissions on organization level.

> the grant_owner_role sets the owner role for the user in the project selected, the permissions that Service Account Administrator owns are also included, so I guess that it's not needed to add it explicitly to the rake task.

@amatas `grant_owner_role` works on project level, and required permissions need to be set on organization level. See above.

File: shared/rakefiles/xk_infra.rake [hunk 1]
```diff
@@ -14,6 +14,8 @@ cloud_admin_organization_permissions = [
   "roles/cloudsupport.admin",
   "roles/orgpolicy.policyAdmin",
   "roles/resourcemanager.organizationAdmin",
+  "roles/iam.serviceAccountAdmin",
+  "roles/securitycenter.admin",
```
[the remark this answers is not in the document]
Comment (natarajaya) [reply shared/rakefiles/xk_infra.rake]: @amatas Replied in thread.

**Comment by stepanstipl** [comment]
> As alternative we can change it to read-only role – SCC Admin Viewer

Yes, I would prefer that.

> @amatas grant_owner_role works on project level, and required permissions need to be set on organization level. See above.

I would consider adding a parallel step for org - same principle - only non-admin tasks for day-to-day ops, and when you need to change smth., you have to take an extra step to elevate your priviledges.

**Comment by natarajaya** [comment]
@amatas @stepanstipl Ok, this PR is now focused around adding single `securitycenter.viewer` role to cloud admin account, which should be sufficient for day-to-day operations.

Will open separate PRs for org-level privilege escalation and enforcement issues.

**Comment by stepanstipl** [comment]
lgtm

**Pull request merged**